client/post-editor/editor-title/style.scss

```diff
@@ -1,10 +1,6 @@
 .editor-title {
```

Comment:
>Is there anything we could simplify markup-wise here?

We'll likely need the relative wrapper for the loading block still, but I'll make another pass to be certain.